hedera-node/configuration/compose/settings.txt

```diff
@@ -1,16 +1,53 @@
-checkSignedStateFromDisk,                      1
-csvFileName,                                   MainNetStats
-csvOutputFolder,                               data/stats
-loadKeysFromPfxFiles,                          0
+                                          #############################
+                                          #    Gossip / Networking    #
+                                          #############################
+
+chatter.useChatter,                            false
 doUpnp,                                        false
-maxOutgoingSyncs,                              1
-reconnect.active,                              1
-reconnect.asyncStreamTimeoutMilliseconds,      60000
-reconnect.reconnectWindowSeconds,              -1
-showInternalStats,                             1
-state.saveStatePeriod,                         300
+maxOutgoingSyncs,                              1                            # differs from mainnet
+numConnections,                                1000
 useLoopbackIp,                                 false
-waitAtStartup,                                 false
-virtualMap.preferredFlushQueueSize,            10000
+
+                                          #############################
+                                          #           State           #
+                                          #############################
+
+jasperDb.iteratorInputBufferBytes,             16777216
+merkleDb.iteratorInputBufferBytes,             16777216
+state.checkSignedStateFromDisk,                true
+state.saveStatePeriod,                         300                          # differs from mainnet
+state.signedStateDisk,                         5
+state.signedStateKeep,                         10
```

Comment:
These two lines are new. Intentional?

Reply:
The default is 3, which is what it used before since it was not specified. If the state used in whatever environment uses this `settings.txt` file is disk constrained, it may cause a problem. Best to check first.

Reply:
Makes sense. I've reverted this back to 3 for this environment.